src/SearchableEntity.php

```diff
@@ -35,11 +58,12 @@ public function getIndexName()
     public function getSearchableArray()
     {
         $context = [
+            'rootEntity'    => $this->entityMetadata->name,
             'fieldsMapping' => $this->entityMetadata->fieldMappings,
         ];
 
-        if ($this->useSerializerGroups) {
-            $context['groups'] = [Searchable::NORMALIZATION_GROUP];
+        if ($this->serializerGroups !== null) {
```

Comment:
This is breaking, the default value should be `[Searchable::NORMALIZATION_GROUP]`.